### ISSUE TITLE
LOGIC changes are not being saved

### ISSUE THREAD

**Issue by normon66** [CCALI/a2jauthor#359]
I have experiencing errors with a counter variable and discovered that it was ultimately because changes to Before logic were not being saved.

Take page _13c1-Property Descriptions_ in the attached interview and see the attached gif. When I make changes and move the focus to another field, it looks as if the changes were saved, but when I reopen the page, it reverted back to the original text. I believe I had originally copy and pasted from somewhere else into this logic field, so my guess is that there is some non-visible character that is throwing off the editor. 

**Can these LOGIC fields just be made plain text, or least have a feature where pasting text removes all non-text characters?** They have been troublesome for years, especially if trying to copy logic from another area or copy code that had highlighted/formatted errors, and forget about cross-browser copy and pasting (I always have to add a middle man by pasting into Notepad and then copying that to ensure all hidden and non-conforming characters). 

For the record, I did save and close the interview and did a Ctrl+F5 on the browser to clear things out.

Possibly related to issue #343 

![logic save](https://user-images.githubusercontent.com/32527754/178780244-76f9d1a4-f208-4e01-a6d5-e67fc32f9c02.gif)

Production A2J v_A2J 10.0.1-2022-07-06_
Chrome v_103.0.5060.114_

[ALS Legal - Leech Lake Petition for Dissolution of Marriage without Children (LL04).zip](https://github.com/CCALI/a2jauthor/files/9104033/ALS.Legal.-.Leech.Lake.Petition.for.Dissolution.of.Marriage.without.Children.LL04.zip)

**Comment by JessicaFrank** [comment]
@normon66 are you able to save the before logic at all? (like deleting the copied info and re-typing it in)

**Comment by normon66** [comment]
@JessicaFrank Yes, selecting and deleting _all_ of the text does trigger a successful save, further indicative of some troublesome, hidden character(s).

**Comment by normon66** [comment]
Actually, it is still finnicky and inconsistent. I was able to delete the BEFORE code once and then manually type some, which saved. 

However, manually editing the same code was still not triggering a save event. and now I am stuck with BEFORE code that I cannot edit nor delete. Editing or deleting AFTER logic works fine.

Looking at the raw xml, I see that there is a `<BR/>` at the end of the BEFORE code, which is likely the culprit. I may end up needing to manually remove this and re-upload the interview, because right now it is unworkable. 

![image](https://user-images.githubusercontent.com/32527754/178789421-066ed64e-13d5-4cb1-a231-79649098acd1.png)
